assets/activation-scripts/activate.d/attach-inplace.bash

```diff
@@ -8,31 +8,36 @@ case "$_flox_shell" in
     echo "export _flox_activate_tracelevel=\"$_flox_activate_tracelevel\";"
     echo "export FLOX_ENV=\"$FLOX_ENV\";"
     echo "export _FLOX_ACTIVATION_STATE_DIR=\"$_FLOX_ACTIVATION_STATE_DIR\";"
-    echo "source '$FLOX_ENV/activate.d/bash';"
+    echo "export _activate_d=\"$_activate_d\";"
+    echo "export _profile_d=\"$_profile_d\";"
```

Comment:
blocking: aren't profile scripts only used in `start.bash`?
```suggestion
```

Reply:
Yes, so it's being passed down unnecessarily, but I'm not sure why that's *blocking*

Reply:
I find these scripts hard to reason about so I don't want us to have misdirection about the flow of information. I would be okay with a followup PR, but I don't want it to live on in the codebase indefinitely, and it's trivial to drop

Reply:
I should have marked it `followup` instead

Reply:
Fixed